### PR TITLE
fix text type bug, now text type don't support

### PR DESCRIPTION
**Pull request by bbbearxyz**
default value

## What type of PR is this? 

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #4538

## What this PR does / why we need it:

### CODE REVIEW

**Review by ouyuanning — APPROVED** [review]

File: pkg/sql/plan/build_util.go [hunk 1]
```diff
@@ -137,6 +136,10 @@ func buildDefaultExpr(col *tree.ColumnTableDef, typ *plan.Type) (*plan.Default,
 		}, nil
 	}
 
+	if typ.GetId() == int32(types.T_blob) {
+		return nil, errors.New(errno.InvalidColumnDefinition, fmt.Sprintf("Type text don't support default value"))
```
Comment (fengttt) [inline pkg/sql/plan/build_util.go:140]: Why?

Comment (bbbearxyz) [reply pkg/sql/plan/build_util.go:140]: because in the db, blob type now is text type, text type don't support default value.

Comment (fengttt) [reply pkg/sql/plan/build_util.go:140]: why is that?   If varchar can have default values, why not text?

**Comment by domingozhang** [comment]
In MySQL, BLOB, TEXT, GEOMETRY or JSON column can't have a default value. I think MO can follow the same rule

**Review by aunjgr — APPROVED** [review]

**Comment by fengttt** [comment]
> 

I don't think so, I dont see any reason not support default value.  Why turning off a feature?

**Comment by fengttt** [comment]
Please update test, supporting default values instead.   Let's close this PR.

**Pull request closed without merging**